Common/Converter.php

```diff
@@ -33,16 +33,11 @@ public static function toAscii(string $str): string
         return $clean;
     }
 
-    public static function formatBytes(int $bytes, int $precision = 2): string
+    public static function formatBytes($size, $precision = 2)
```

Comment:
```suggestion
    public static function formatBytes(int $bytes, int $precision = 2): string
```
You should keep the type hinting (int, string)!
And $bytes is more detailed then $size !